### PR TITLE
CI testing

### PR DESCRIPTION
**Pull request by philbucher**
not for merge, only for testing

FYI @oberbichler @roigcarlo

### CODE REVIEW

**Comment by roigcarlo** [comment]
@philbucher What is this about Oo?

**Comment by philbucher** [comment]
just testing the actions
in particular the workflow is only executed if sth is pushed to the apps that are being compiled
E.g. here I change sth in the CoSimApp (which is not being compiled) and it should not (and actually does not :D) trigger the build

**Comment by roigcarlo** [comment]
Well it has a quite big filter, I think its normal: 

```yml
on:
  pull_request:
    paths:
    - '.github/**'
    - 'cmake_modules/**'
    - 'scripts/**'
    - 'CMakeLists.txt'
    - 'external_libraries/**'
    - 'kratos/**'
    - 'applications/FluidDynamicsApplication/**'
    - 'applications/StructuralMechanicsApplication/**'
    - 'applications/DEMApplication/**'
    - 'applications/MeshMovingApplication/**'
    - 'applications/CSharpWrapperApplication/**'
    - 'applications/MetisApplication/**'
    - 'applications/TrilinosApplication/**'
    - 'applications/ExternalSolversApplication/**'
```

**Comment by philbucher** [comment]
> Well it has a quite big filter, I think its normal:

yes I added this
I just want to make sure its working ;)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -17,6 +17,11 @@ on:
     - 'applications/MetisApplication/**'
     - 'applications/TrilinosApplication/**'
     - 'applications/ExternalSolversApplication/**'
+    - 'applications/ContactStructuralMechanicsApplication/**'
+    - 'applications/MeshingApplication/**'
+    - 'applications/SwimmingDEMApplication/**'
+  schedule:
+    - cron:  '0 1 * * *'
```
Comment (philbucher) [inline .github/workflows/ci.yml:24]: this is basically the nightly build

**Review by oberbichler — APPROVED** [review]

**Comment by philbucher** [comment]
ok now we have a nightly build :)

**Comment by roigcarlo** [comment]
Which is the time limit for the night builds in github? if its high (or the cost to expand it is not elevate) maybe we can retire the amazon machine

**Comment by philbucher** [comment]
There is no limit from what I understood (@oberbichler please correct me if I am wrong)
by default jobs are being killed after 360m but I think this could be changed
what I pushed here I have already working in a small repo of mine, works fine so far

> Which is the time limit for the night builds in github? if its high (or the cost to expand it is not elevate) maybe we can retire the amazon machine

That could work! I have some more ideas that I will try beginning of next year, then we can see further
If you are busy otherwise, then I suggest to not focus on the AWS build until we see how this here goes

**Comment by oberbichler** [comment]
[Usage limits](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/about-github-actions#usage-limits)

**Comment by roigcarlo** [comment]
Mmmm 6h runtime. Well, in that case is time to say goodby to amazon I think :D

**Pull request merged**